<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantCurso</class>
 <widget class="QDialog" name="MantCurso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento de Cursos</string>
  </property>
  <widget class="QComboBox" name="cmbTipo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Clase</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laboratorio</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtCodigo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNumClase">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDias">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtInstructor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAtras">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSiguiente">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCodigo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFecha">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInstructor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDias">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Dias:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumClase">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clase/Lab:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFecha">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>##/##/####</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
